src/equation_of_state/MOM_EOS_UNESCO.F90

```diff
@@ -205,49 +219,37 @@ subroutine calculate_spec_vol_array_UNESCO(T, S, pressure, specvol, start, npts,
   real :: s12  ! The square root of salinity [PSU1/2]l553
```

Comment:
What is the "1553" doing at the end of this line?  It looks lie a typo that should be removed.

Reply:
🤷‍♂️ No idea where that came from. Fixed.